package.json

```diff
@@ -5,14 +5,20 @@
   "repository": "git@github.com:superfaceai/station.git",
   "author": "Superface Team",
   "scripts": {
+    "build_reporter": "tsc src/reporter/*.ts --outDir dist",
     "tools:test": "jest -c jest.tools.config.js",
     "tools:lint": "eslint ./",
     "tools:lint:fix": "yarn tools:lint --fix",
     "tools:format": "prettier -c ./",
     "tools:format:fix": "prettier --write ./",
     "prepush": "yarn tools:lint:fix && yarn tools:format:fix",
     "test": "dotenv -e .env.example -- jest",
-    "test:record": "SUPERFACE_LIVE_API='*' dotenv -e .env -e .env.example -- jest -u",
+    "test:with_new_traffic": "USE_NEW_TRAFFIC=true yarn test",
+    "test:update_traffic": "UPDATE_TRAFFIC=true yarn test",
+    "test:record": "SUPERFACE_LIVE_API='*' dotenv -e .env -e .env.example -- jest -c jest.live.config.js",
+    "test:record:dev": "TEST_ENV='dev' yarn test:record",
+    "test:record:prod": "TEST_ENV='prod' yarn test:record --group=live/safe",
+    "test:update_snapshots": "yarn test -u",
```

Comment:
issue: I have no clue when to use what :(

Reply:
I can add it to readme in separate PR, I already describe them in notion (see [here](https://www.notion.so/superface/Overview-of-station-scripts-814cec1d63b947fb92c2248c7e973c9e))

Reply:
Since this is a public repository, it should be here in the repository.